### PR TITLE
8.0-rc3 Release Notes

### PR DESCRIPTION
**Pull request by hythloda**
[Newest version of rendered notes here](https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/60778253:id/distributions/gradle-8.0-all.zip!/gradle-8.0-20230126001600%2B0000/docs/release-notes.html)

### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [x] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [x] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [x] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Gradle Core Team Checklist
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation

### CODE REVIEW

**Comment by hythloda** [comment]
Screenshot 2023-01-25 at 9 27 56 AM
I changed the h4 to look different from h3

**Comment by hythloda** [comment]
@bot-gradle test BD

**Comment by hythloda** [comment]
Screenshot 2023-01-25 at 7 11 35 PM

File: build-logic/documentation/src/main/groovy/gradlebuild/docs/ReleaseNotesTransformer.java [hunk 1]
```diff
@@ -160,13 +160,33 @@ private void addTOC(Document document) {
         tocSection.append("<h2>Table Of Contents</h2>");
         Element toc = tocSection.append("<ul class='toc'/>").children().last();
 
-        for (Element topic : document.body().select(".topic")) {
-            Element topicHeading = topic.select("h2").first();
-            String name = topicHeading.text();
-            String anchor = topicHeading.attr("id");
-            toc.append("<li><a/></li>").children().last().select("a").first().text(name).attr("href", "#" + anchor);
+        for (Element topic : document.body().select("h2")) {
+            //Element topicHeading = topic.select("h2,h3").first();
```
Comment (eskatos) [inline build-logic/documentation/src/main/groovy/gradlebuild/docs/ReleaseNotesTransformer.java:164]: :lipstick: I would drop this dead commented code

File: build-logic/documentation/src/main/groovy/gradlebuild/docs/ReleaseNotesTransformer.java [hunk 2]
```diff
@@ -85,7 +85,7 @@ private Reader transform(Reader in) throws IOException {
 
         wrapH2InSectionTopic(document);
         addAnchorsForHeadings(document);
-        document.body().prepend("<h3 class='releaseinfo'>Version @version@</h3>");
+        //document.body().prepend("<h3 class='releaseinfo'>Version @version@</h3>");
```
Comment (eskatos) [inline build-logic/documentation/src/main/groovy/gradlebuild/docs/ReleaseNotesTransformer.java:88]: :lipstick: I would drop this dead commented code

**Review by eskatos — APPROVED** [review]

**Comment by hythloda** [comment]
@bot-gradle test and merge

**Comment by hythloda** [comment]
@bot-gradle test this

**Pull request merged**